cirq/ops/parity_gates.py

```diff
@@ -124,7 +134,24 @@ def __repr__(self) -> str:
 class YYPowGate(eigen_gate.EigenGate,
                 gate_features.TwoQubitGate,
                 gate_features.InterchangeableQubitsGate):
-    """The Y-parity gate, possibly raised to a power."""
+    """The Y-parity gate, possibly raised to a power.
+
+        At exponent=1, this gate implements the following unitary:
+
+        Y⊗Y = [ 0  0  0 -1]
+              [ 0  0  1  0]
+              [ 0  1  0  0]
+              [-1  0  0  0]
+
+        The YY**t gate implements the following unitary:
+
+        (Y⊗Y)^t = [cos(y) . . isin(y)]
```

Comment:
Let's do something similar to https://github.com/quantumlib/Cirq/blob/144e1ebdd6196cf6c8ea8e6f497dbb947bf57142/cirq/ops/common_gates.py#L230